packages/wrangler/src/index.tsx

```diff
@@ -494,6 +494,14 @@ export async function main(argv: string[]): Promise<void> {
         );
       }
 
+      if (config.site?.["entry-point"]) {
+        console.warn(
+          "Deprecation notice: The `site.entry-point` config field is no longer used.\n" +
+            "The entry-point is specified via the command line (see `wrangler dev --help`).\n" +
```

Comment:
Feels like a pseudo example would be more helpful here than asking them to see the help text. IMO ofc, your call. 
```suggestion
            "The entry-point should be specified via the command line as `wrangler dev path/to/entry` or the `build.upload.main` config field.\n" +
```

Reply:
Yeah - cool. That sounds like a good idea. I'll update

Reply:
Actually in the case of `dev` the script must be passed, so it is not possible to use `build.upload.main` here.